src/mapml-viewer.js

```diff
@@ -986,7 +989,256 @@ export class HTMLMapmlViewerElement extends HTMLElement {
       }
     });
   }
+  matchMedia(query) {
+    // useful features for maps: prefers-color-scheme, prefers-lang, projection, zoom, extent
+    const parsedQuery = parseMediaQueryList(query);
+
+    // less obviously useful: aspect-ratio, orientation, (device) resolution, overflow-block, overflow-inline
+
+    const map = this;
+    const features = {
+      'prefers-lang': {
+        type: 'discrete',
+        get values() {
+          return [navigator.language.substring(0, 2)];
```

Comment:
language subtag may not always be of 2 characters i.e. https://datatracker.ietf.org/doc/html/rfc5646#appendix-A. Would it instead be better to use `.split('-')[0]`?